orc8r/cloud/go/services/bootstrapper/servicers/registration/helpers.go

```diff
@@ -0,0 +1,33 @@
+package registration
+
+import (
+	"math/rand"
+	"time"
+
+	"magma/orc8r/cloud/go/services/bootstrapper"
+	"magma/orc8r/lib/go/protos"
+)
+
+func nonceToToken(nonce string) string {
+	return bootstrapper.TokenPrepend + nonce
+}
+
+func nonceFromToken(token string) string {
+	return token[len(bootstrapper.TokenPrepend):]
+}
+
+func isTokenExpired(info *protos.TokenInfo) bool {
+	return time.Now().Before(time.Unix(0, int64(info.Timeout.Nanos)))
+}
+
+// ========================================================================= //
```

Comment:
Nit: I'm not a fan of floating "big block" comments. Instead, let's attach this comment directly to the function, as a doc comment